corehq/apps/importer/views.py

```diff
@@ -29,6 +29,18 @@ def render_error(request, domain, message):
 
 @require_can_edit_data
 def excel_config(request, domain):
+    """
+    Step one of three.
+
+    This post uploads the file and is when the user enters:
+
+    named_columns:
+        Whether or not the first row of the excel sheet contains
+        header strings for the columns. This defaults to True and
+        should potentially not be an option as it is always used
+        due to how important it is to see column headers
+        in the rest of the importer.
+    """
```

Comment:
Oh, I thought there were going to be more in the list but the string just reads strangely..